app/homepage/page.tsx

```diff
@@ -1,116 +1,117 @@
-'use client'
-import React, { useState } from 'react';
-import Image from 'next/image';
-import logo from '../../public/images/logo.png';
-import { ChevronLeft, Trash2 } from 'lucide-react';
-import Link from 'next/link';
-import { Dialog, DialogTrigger, DialogContent, DialogTitle } from "@/components/ui/dialog";
+  'use client'
+  import React, { useState } from 'react';
+  import Image from 'next/image';
+  import logo from '../../public/images/logo.png';
+  import { ChevronLeft, Trash2 } from 'lucide-react';
+  import Link from 'next/link';
+  import { Dialog, DialogTrigger, DialogContent, DialogTitle } from "@/components/ui/dialog";
 
-import { Input } from '@/components/ui/input';
-import { Button } from '@/components/ui/button';
-import paper from '../../public/images/paper.png';
+  import { Input } from '@/components/ui/input';
+  import { Button } from '@/components/ui/button';
+  import paper from '../../public/images/paper.png';
 
-const LandingPage = () => {
-  const [projectName, setProjectName] = useState('');
-  const [projects, setProjects] = useState([]);
-  
-  const [isDialogOpen, setIsDialogOpen] = useState(false);
-
-  const handleSubmit = (event: { preventDefault: () => void; }) => {
-    event.preventDefault();
-
-    const newProject = {
-        id: projectName,
+  const LandingPage = () => {
+    const [projectName, setProjectName] = useState('');
```

Comment:
add object types to variables/constant

eg. [projectName, setProjectName] = useState<string>("");

^Less headache in the future

Reply:
[51832115154a23aca88249d3dad51b48d767ec99](https://github.com/21ftt1750/tododo/pull/2/commits/51832115154a23aca88249d3dad51b48d767ec99)

fixed